contracts/Overtime/SportsAMMV2.sol

```diff
@@ -118,6 +118,17 @@ contract SportsAMMV2 is Initializable, ProxyOwned, ProxyPausable, ProxyReentranc
     // stores tickets per game
     mapping(bytes32 => AddressSetLib.AddressSet) internal ticketsPerGame;
 
+    address public liveTradingProcessor;
+
+    struct TradeDataInternal {
```

Comment:
I added similar struct on the ISportsAMMV2 interface contract. 
Regarding contract size there is no difference, so we can go we either choice.